shared/src/ledger/ibc/vp/client.rs

```diff
@@ -571,7 +573,16 @@ where
             .add(gas)
             .map_err(|_| Ics02Error::implementation_specific())?;
         match header {
-            Some(h) => Ok(TmConsensusState::from(h).wrap_any()),
+            Some(h) => Ok(TmConsensusState {
+                root: CommitmentRoot::from_bytes(h.hash.as_slice()),
+                timestamp: h.time.try_into().unwrap(),
+                next_validators_hash: h
```

Comment:
I'm not sure I understand the proposal here.

Reply:
something like
```rust
next_validators_hash: crate::tendermint::Hash::Sha256(*h.next_validators_hash),
```
to avoid going via a vec and having to handle a result, though this way looks clunky

Reply:
Ah, I see. No, I'm good with that.